ci/build-all-defconfigs.sh

```diff
@@ -62,25 +62,20 @@ if [ -n "$DL_DIR" ]; then
 fi
 
 for i in ${DEFCONFIGS[@]}; do
-        rm -rf output/*
-        op-build $i
-        echo 'BR2_CCACHE=y' >> output/.config
-        echo "BR2_CCACHE_DIR=\"$CCACHE_DIR\"" >> output/.config
-        echo 'BR2_CCACHE_INITIAL_SETUP=""' >> output/.config
-
-        op-build olddefconfig
-        op-build
+	export O=${OUTDIR}
+        op-build O=$O $i
+	./buildroot/utils/config --file $O/.config --set-val BR2_CCACHE y
+        ./buildroot/utils/config --file $O/.config --set-str BR2_CCACHE_DIR $CCACHE_DIR
+        op-build O=$O olddefconfig
+        op-build O=$O
         r=$?
-
-        if [ ${BUILD_INFO} = 1 ] && [ $r = 0 ]; then
-                op-build legal-info
-                mv output/legal-info ${OUTDIR}/$i-legal-info
-        fi
-
-        mkdir ${OUTDIR}/$i-images
-        mv output/images/* ${OUTDIR}/$i-images/
```

Comment:
I think we lose something here. Instead of moving the output images of each platform to
`${OUTDIR}/platform_name-images/` we leave everything in the `${OUTDIR}` - which is fine for building but not if we want to grab the images.
We also don't `rm -rf` the output directory (now `${OUTDIR}`) so each platform will build on top of itself I think.

Reply:
yeah, fair enough on cleaning things out. I'll fix that out.

We should be passing through `pwd` to the container, so we'll end up with a bunch of output dirs in `pwd` with all the build artifacts. This is either a feature or a bug I guess, depending on how much disk space you have.